esphome/components/addressable_light_digital_display/display.py

```diff
@@ -0,0 +1,86 @@
+import copy
+
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import display, light
+from esphome.const import (
+    CONF_ID,
+    CONF_LAMBDA,
+    CONF_ADDRESSABLE_LIGHT_ID,
+    CONF_LIGHT_ID,
+    CONF_UPDATE_INTERVAL,
+)
+
+CODEOWNERS = ["@daweizhangau"]
+
+CONF_LED_MAP = "led_map"
+CONF_REVERSE = "reverse"
+
+addressable_light_ns = cg.esphome_ns.namespace("addressable_light_digital_display")
+AddressableLightDisplay = addressable_light_ns.class_(
+    "DigitalDisplay", cg.PollingComponent
+)
```

Comment:
Please make sure the variables in python match the actual names in c++.
I was very confused when reading this
```suggestion
addressable_light_digital_display_ns = cg.esphome_ns.namespace("addressable_light_digital_display")
DigitalDisplay = addressable_light_digital_display_ns.class_(
    "DigitalDisplay", cg.PollingComponent
)
```